samza-yarn/src/test/java/org/apache/samza/job/yarn/TestYarnFaultDomainManager.java

```diff
@@ -60,22 +60,22 @@ public class TestYarnFaultDomainManager {
 
   private final NodeReport nodeReport1 = createNodeReport(hostName1, 1, NodeState.RUNNING, "httpAddress1",
           rackName1, 1, 1, 2, 1, 2,
-          "", 60L, null);
+          "", 60L);
```

Comment:
Is this change for YARN 3.3.4? I would suggest leave it out since we are not changing samza-yarn module for YARN 3.3.4.